packages/exchange-core/src/index.ts

```diff
@@ -5,3 +5,4 @@ export { Ask } from './Ask';
 export { Product } from './Product';
 export { DeviceVintage } from './DeviceVintage';
 export { Operator } from './Operator';
+export * from './order-book-order.dto';
```

Comment:
Let not move those DTO objects to core project. They are specific for exchange and clients and meant for the data serialization/deserialization.

I think it's fine to have either a copy of DTO object in the UI or create exchange-client package and put it there.